CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 5.0
+
+## Removed 
+
+## 4.6
+
+Deprecated […] methods, use loginUser() from Symfony instead https://symfony.com/doc/5.4/testing.html#logging-in-users-authentication
```

Comment:
Thanks for your input!

`WebTestCase::isSuccessful` could be replaced by [`assertResponseIsSuccessful`](https://symfony.com/doc/4.4/testing.html#response-assertions) which exist since Symfony 4.4.

`DatabaseToolCollection` comes from the other bundle: https://github.com/liip/LiipTestFixturesBundle, which should work with Symfony 6.6.